development/circular-deps.jsonc

```diff
@@ -43,14 +43,6 @@
     "ui/components/app/assets/asset-list/native-token/native-token.tsx",
     "ui/components/app/assets/asset-list/native-token/use-native-token-balance.ts"
   ],
-  [
-    "ui/components/app/assets/token-list/token-list.tsx",
-    "ui/components/app/assets/util/calculateTokenBalance.ts"
-  ],
-  [
-    "ui/components/app/assets/token-list/token-list.tsx",
-    "ui/components/app/assets/util/calculateTokenFiatAmount.ts"
-  ],
```

Comment:
@HowardBraham @dbrans @MetaMask/extension-security-team I believe these circular dependency removals flagged you on this PR.

Do you folks see any issues with their removal?

Reply:
there are never issues with removal!

Reply:
We put this check in because we're trying to remove and get it down to zero.  I'll always approve removal.  If you add new circular dependencies... well then we'll have a conversation.